### PR TITLE
#49candi-meter-14

### PR DESCRIPTION
**Pull request by fatmaalzharaa**

### CODE REVIEW

File: candi-meter/candi-meter-ui/.vscode/extensions.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+  // For more information, visit: https://go.microsoft.com/fwlink/?linkid=827846
```
Comment (Tasneem-Gad) [inline candi-meter/candi-meter-ui/.vscode/extensions.json:2]: are you need this

File: candi-meter/candi-meter-ui/.vscode/launch.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+      "preLaunchTask": "npm: start",
+      "url": "http://localhost:4200/"
+    },
+    {
+      "name": "ng test",
+      "type": "chrome",
+      "request": "launch",
+      "preLaunchTask": "npm: test",
+      "url": "http://localhost:9876/debug.html"
+    }
```
Comment (Tasneem-Gad) [inline candi-meter/candi-meter-ui/.vscode/launch.json:18]: are you need this

File: candi-meter/candi-meter-ui/.vscode/tasks.json [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    {
+      "type": "npm",
+      "script": "test",
+      "isBackground": true,
+      "problemMatcher": {
+        "owner": "typescript",
+        "pattern": "$tsc",
+        "background": {
+          "activeOnStart": true,
+          "beginsPattern": {
```
Comment (Tasneem-Gad) [inline candi-meter/candi-meter-ui/.vscode/tasks.json:32]: are you need this

**Review by Tasneem-Gad — CHANGES_REQUESTED** [review]

File: candi-meter/candi-meter-ui/.vscode/extensions.json (hunk 1 above)
Comment (fatmaalzharaa) [reply candi-meter/candi-meter-ui/.vscode/extensions.json:2]: no. i will remove now

**Pull request merged**